Caltech-IPAC/firefly
# Firefly-499: fixed actions being lost in WebsocketConnector

Firefly-499: fixed actions being load in WebsocketConnector
   - created test to send many table to Firefly
   - fixed a problem with unnecessary HiPS message

Fixes: Firelfy-509

https://jira.ipac.caltech.edu/browse/FIREFLY-499

_To test:_ 
1. use: https://irsawebdev9.ipac.caltech.edu/firefly-499-api-losing-tables/firefly/test/tests-viewer.html
1. go to test 5 and click
1. a firefly should load with all ten tables.
1. delete the firefly tab and repeat several times to make sure it always gets 10 tables.
